src/firefly/js/charts/ui/ChartsContainer.jsx

```diff
@@ -98,9 +98,11 @@ export const ChartsContainer = (props)  =>{
     const {chartId, expandedMode} = props;
 
     // eslint-disable-next-line react-hooks/rules-of-hooks
-    return expandedMode || chartId || !allowPinnedCharts() ?
-        <DefaultChartsContainer {...props}/>:
-        <PinnedChartPanel {...props}/> ;
+    return <PinnedChartPanel {...props}/>;
+
+    //return expandedMode || chartId || !allowPinnedCharts() ?
+       // <DefaultChartsContainer {...props}/>:
+        //<PinnedChartPanel {...props}/> ;
```

Comment:
I don't know the reason why this is needed.  As is, it breaks expanded mode.  Maybe you can show me the case when this is not working.